src/Cake.Svn/Info/SvnInfoResult.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Cake.Svn.Internal.Extensions;
+
+namespace Cake.Svn.Info
+{
+    /// <summary>
+    /// Result for <see cref="SvnInfo"/>.
+    /// </summary>
+    public sealed class SvnInfoResult
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SvnInfoResult"/> class.
+        /// </summary>
+        public SvnInfoResult(
+            Guid repositoryId, 
+            Uri repositoryRoot, 
+            string lastChangedAuthor,
+            long revision,
+            Uri uri,
+            string path,
+            string fullPath,
+            SvnKind nodeKind)
+        {
+            repositoryRoot.NotNull(nameof(repositoryRoot));
+            lastChangedAuthor.NotNull(nameof(lastChangedAuthor));
+            uri.NotNull(nameof(uri));
+            path.NotNull(nameof(path));
+            fullPath.NotNull(nameof(fullPath));
+
+            RepositoryId = repositoryId;
+            RepositoryRoot = repositoryRoot;
+            LastChangedAuthor = lastChangedAuthor;
+            Revision = revision;
+            Uri = uri;
+            Path = path;
+            FullPath = fullPath;
+            NodeKind = nodeKind;
+        }
+
+        /// <summary>
+        /// Gets the uuid of the repository (if available). Otherwise Guid.Empty.
+        /// </summary>
+        public Guid RepositoryId { get; }
+
+        /// <summary>
+        ///  Gets the repository root Uri; ending in a '/'.
+        /// </summary>
+        /// <remarks>
+        /// The unmanaged api does not add the '/' at the end, but this makes using System.Uri hard.
```

Comment:
This is an implementation detail and not of interested for the consumer of the API. Should be documented where the mapping between unmanaged API and property value is made